workflows/illumina.nf

```diff
@@ -1,12 +1,8 @@
-////////////////////////////////////////////////////
-/* --         LOCAL PARAMETER VALUES           -- */
-////////////////////////////////////////////////////
-
-params.summary_params = [:]
-
-////////////////////////////////////////////////////
-/* --          VALIDATE INPUTS                 -- */
-////////////////////////////////////////////////////
+/*
```

Comment:
The more I see them the more I like this new comments style 😍

Reply:
💥